.clang-format

```diff
@@ -0,0 +1,156 @@
+---
+# This file is copied from cuDF repo.
```

Comment:
Okay, so link to it?

https://github.com/rapidsai/cudf/blob/branch-0.17/cpp/.clang-format